portfolio/src/main/java/com/google/sps/servlets/GetNickname.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+
+
+public class GetNickname {
```

Comment:
Better to rename it to `UserStore` or `UserDatabase` or something similar. We don't usually name a class with `GetXXX` or `SetXXX`.

Reply:
Please also a javadoc for this class.